src/client/clientErrorHandler.ts

```diff
@@ -23,6 +28,11 @@ export class ClientErrorHandler implements ErrorHandler {
 
   closed(): CloseAction {
     this.restarts.push(Date.now());
+    const heapProfileGlob = new glob.GlobSync(`${path.resolve(__dirname, '..')}/java_*.hprof`);
+    if (heapProfileGlob.found.length) {
+      showOOMMessage();
```

Comment:
I was originally thinking that the user could read the heap dump files to help find memory leaks. However, since the files are a binary format, I think its a good idea to clean them up here.